easybuild/easyconfigs/h/HDF/HDF-4.2.13-GCCcore-6.4.0-libjpeg-turbo-1.5.3.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'HDF'
+version = '4.2.13'
```

Comment:
note to self: there's a newer version of `HDF`, see https://portal.hdfgroup.org/display/support/Download+HDF4